amalgamate.py

```diff
@@ -386,7 +386,7 @@ def rewrite_imports(name, pkg, order, imps):
             for n in a.names:
                 p, dot, m = n.name.rpartition('.')
                 if p == pkg and m in order:
-                    msg = ('Cannot amalgamate almagate import of '
+                    msg = ('Cannot amalgamate amalgam import of '
```

Comment:
Yes, the sentence is definitely a bit awkward. There are a number of sentences that felt a little bit off, but I didn't want to risk offending someone with my own grammatical style, or, worse yet, ruin the sentence by introducing technical inaccuracies.

Now, that said, I don't know the code well enough to authoritatively say that the sentence as is (with the word amalgam instead of almagate) is not actually accurate. Is the code performing a super-amalgamate function, such that it is amalgamating an amalgamated bundle of code (just writing that out makes me realize how easy it is to misspell amalgamate 😉 )? If so, then the sentence might be better left alone. If you believe the sentence is more accurate with that word removed, I definitely don't mind including that in this PR. I just don't want to make